api/src/main/java/jakarta/xml/bind/ContextFinder.java

```diff
@@ -10,6 +10,9 @@
 
 package jakarta.xml.bind;
 
+import static jakarta.xml.bind.JAXBContext.JAXB_CONTEXT_FACTORY;
+import static java.util.logging.Level.FINE;
```

Comment:
Do not use static imports

Reply:
> Do not use static imports

Why not? It's a fully legal Java construct. I can change them, but just wonder why.

I know that Sun had a convention to fully qualify everything (like fully typing 'java.lang.String`). Is that related to this?

Reply:
Just because `_` or `$` prefixed strings are valid java identifiers, should we be using that? It's not related to Sun standard, it's related to readability of the source - when one sees `JAXB_CONTEXT_FACTORY` there, how exactly can he find out where it is defined (ie in vi, emacs, notepad or any simple text editor - yes, I'm using these occasionally)

Reply:
If you want to see where it's defined, you can look at the imports?

Otherwise it's a little arbitrary perhaps  to want `JAXBContext.JAXB_CONTEXT_FACTORY` and not `jakarta.xml.bind.JAXBContext.JAXB_CONTEXT_FACTORY`?

I mean, either fully quality because you're using vi (yes, I use it too), or don't qualify because you look at the imports or are using an IDE.

But no worries, I'll remove the static imports.